tempesta_fw/sched/tfw_sched_hash.c

```diff
@@ -29,84 +39,189 @@
  * Temple Place - Suite 330, Boston, MA 02111-1307, USA.
  *
  */
+#include <linux/hash.h>
 #include <linux/module.h>
 
+#include "http_msg.h"
 #include "log.h"
 #include "sched.h"
-#include "http_msg.h"
 
 MODULE_AUTHOR(TFW_AUTHOR);
 MODULE_DESCRIPTION("Tempesta hash-based scheduler");
-MODULE_VERSION("0.0.1");
+MODULE_VERSION("0.0.2");
```

Comment:
I referenced a link to the versioning scheme in one of my previous reviews. Please fix the version.